src/components/Header/Header.js

```diff
@@ -1,25 +1,49 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
 import style from './style.scss';
 
 export default class Header extends Component {
   state = {
     isAdmin: false,
     logIn: false,
-    userName: ''
+    userName: '',
+    anchorEl: null
```

Comment:
What this property for? Name it properly.
Also working with DOM elements directly doesn't feel natural in React. Try to think of another approach.